### ISSUE TITLE
Attributes not defined: <SN_FCC_Buck> <SN_FCC_MPPT>

### ISSUE THREAD

**Issue by vplin** [fnoorian/CloudForecast15#4]
I just reviewed the source code and found these two attributes are not defined before use. 
Is .open() method necessary for the drivers?

**Comment by fnoorian** [comment]
No. This is a relic from the time that we had assigned a certain Arduino board (with a certain serial number) to Buck or Boost. Please remove the open() methods.

By the way, can you open issues under the Free-buck-boost when they are related to that code instead of here?

**Comment by vplin** [comment]
Ok, sure!
